traffic_ops/traffic_ops_golang/crconfig/servers.go

```diff
@@ -361,7 +361,6 @@ order by dsr.set_number asc
 		for _, dsName := range dses {
 			dsInfList, ok := dsInfs[string(dsName)]
 			if !ok {
-				log.Warnln("Creating CRConfig: deliveryservice " + string(dsName) + " has no regexes, skipping")
```

Comment:
We only want to remove this warning for topology-based delivery services. Other DSes should still have the warning.

Reply:
ah, this answers my question on the related Issue

Reply:
Done.